src/ngraph/runtime/cpu/cpu_emitter.cpp

```diff
@@ -4621,15 +4621,17 @@ namespace ngraph
             template <>
             void CPU_Emitter::EMITTER_DECL(ngraph::op::Quantize)
             {
-                auto quantize = static_cast<const ngraph::op::Dequantize*>(node);
+                auto quantize = static_cast<const ngraph::op::Quantize*>(node);
                 writer << "reference::quantize(";
                 writer << "            " << args[0].get_name() << ",\n";
                 writer << "            " << args[1].get_name() << ",\n";
                 writer << "            " << args[2].get_name() << ",\n";
                 writer << "            " << out[0].get_name() << ",\n";
                 writer << "            {" << join(args[0].get_shape()) << "},\n";
                 writer << "            {" << join(args[1].get_shape()) << "},\n";
-                writer << "            {" << join(quantize->get_axes()) << "});\n";
+                writer << "            {" << join(quantize->get_axes()) << "},\n";
+                writer << "            static_cast<op::Quantize::RoundMode>("
+                       << static_cast<int>(quantize->get_round_mode()) << "));\n";
```

Comment:
I'm not positive this is the correct way to do this and also not sure we are hitting this case with unit tests.  Can you comment on this @nbpatel?